### PR TITLE
docs: restructure README header

### PR DESCRIPTION
**Pull request by sherwinski**
This PR restructures the header section of the README, adds more badges, and makes some edits to the text style. The header is wrapped in a tag that will exclude it from being displayed on the [imgix libraries](https://docs.imgix.com/libraries) page.

drift-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,30 @@
...
-    <td align="center"><a href="https://github.com/frederickfogerty"><img src="https://avatars0.githubusercontent.com/u/615334?v=4" width="100px;" alt="Frederick Fogerty"/><br /><sub><b>Frederick Fogerty</b></sub></a><br /><a href="https://github.com/imgix/drift/commits?author=frederickfogerty" title="Code">💻</a> <a href="https://github.com/imgix/drift/commits?author=frederickfogerty" title="Documentation">📖</a> <a href="#maintenance-frederickfogerty" title="Maintenance">🚧</a></td>
-    <td align="center"><a href="http://jayeb.com"><img src="https://avatars2.githubusercontent.com/u/609840?v=4" width="100px;" alt="Jason Eberle"/><br /><sub><b>Jason Eberle</b></sub></a><br /><a href="https://github.com/imgix/drift/commits?author=jayeb" title="Code">💻</a> <a href="https://github.com/imgix/drift/commits?author=jayeb" title="Documentation">📖</a> <a href="#maintenance-jayeb" title="Maintenance">🚧</a></td>
-  </tr>
-</table>
-
-<!-- ALL-CONTRIBUTORS-LIST:END -->
-
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
-
 <a name="installation"></a>
```
Comment (jayeb) [inline README.md]: These `<a name="...">` elements aren't necessary, are they?

Comment (sherwinski) [reply README.md]: No, I don't think so. There are probably other repos where they need to be removed from as well.

File: README.md [hunk 2]
```diff
@@ -207,23 +188,35 @@ setInterval(function() {
...
+    <td align="center"><a href="https://github.com/frederickfogerty"><img src="https://avatars0.githubusercontent.com/u/615334?v=4" width="100px;" alt="Frederick Fogerty"/><br /><sub><b>Frederick Fogerty</b></sub></a><br /><a href="https://github.com/imgix/drift/commits?author=frederickfogerty" title="Code">💻</a> <a href="https://github.com/imgix/drift/commits?author=frederickfogerty" title="Documentation">📖</a> <a href="#maintenance-frederickfogerty" title="Maintenance">🚧</a></td>
+    <td align="center"><a href="http://jayeb.com"><img src="https://avatars2.githubusercontent.com/u/609840?v=4" width="100px;" alt="Jason Eberle"/><br /><sub><b>Jason Eberle</b></sub></a><br /><a href="https://github.com/imgix/drift/commits?author=jayeb" title="Code">💻</a> <a href="https://github.com/imgix/drift/commits?author=jayeb" title="Documentation">📖</a> <a href="#maintenance-jayeb" title="Maintenance">🚧</a></td>
+  </tr>
+</table>
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
```
Comment (jayeb) [inline README.md]: This section needs to be wrapped in `<!-- ix-docs-ignore -->`.

File: README.md [hunk 3]
```diff
@@ -207,23 +188,37 @@ setInterval(function() {
...
 
-<a name="browser-support"></a>
-
 ## Browser Support
 
 We support the latest version of Google Chrome (which [automatically updates](https://support.google.com/chrome/answer/95414) whenever it detects that a new version of the browser is available). We also support the current and previous major releases of desktop Firefox, Internet Explorer, and Safari on a rolling basis. Mobile support is tested on the most recent minor version of the current and previous major release for the default browser on iOS and Android (e.g., iOS 9.2 and 8.4). Each time a new version is released, we begin supporting that version and stop supporting the third most recent version.
 
-<a name="meta"></a>
+<!-- ix-docs-ignore -->
+## Contributors ✨
```
Comment (jayeb) [inline README.md]: Will you pull the `## Contributors` header out of the comment here so it appears along with the "This project follows..." paragraph at the bottom?

File: README.md [hunk 4]
```diff
@@ -1,43 +1,30 @@
-![imgix logo](https://assets.imgix.net/imgix-logo-web-2014.pdf?page=2&fm=png&w=200&h=200)
+<!-- ix-docs-ignore -->
+![imgix logo](https://assets.imgix.net/sdk-imgix-logo.svg)
+
+`drift-zoom` adds easy "zoom on hover" functionality to your site's images, all with lightweight, no-dependency JavaScript.
```
Comment (jayeb) [inline README.md]:
```suggestion
Drift adds easy "zoom on hover" functionality to your site's images, all with lightweight, no-dependency JavaScript.
```

**Pull request merged**